src/interfaces/reactNativeInterface.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  create(styleHash) {
```

Comment:
This could be a hash or an array, correct? Perhaps update the variable name to reflect this?